src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.spring.config;
+
+import java.util.Properties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+    @Override
+    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+        auth.inMemoryAuthentication()
+                .withUser("maks@gmail.com")
+                .password(getPassWordEncoder().encode("1234"))
+                .roles("USER");
+    }
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .authorizeRequests()
+                .anyRequest().authenticated()
+                .and()
+                .formLogin().permitAll()
+                .and()
+                .httpBasic()
+                .and()
+                .csrf().disable();
+    }
+
+    public InMemoryUserDetailsManager inMemoryUserDetailsManager() {
+        Properties users = new Properties();
+        users.put("alice.smith@gmail.com",
+                getPassWordEncoder().encode("12345678") + ",ROLE_USER,enabled");
+        users.put("bob.brown@gmail.com",
+                getPassWordEncoder().encode("qwertyuiop") + ",ROLE_USER,enabled");
+        return new InMemoryUserDetailsManager(users);
+    }
+
+    @Bean
+    public PasswordEncoder getPassWordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
```

Comment:
lets move this method to another configuration class - that way you will be able to autowire  encoder into this class